src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -1103,20 +1111,7 @@ export class EditorHistoryEntry extends EditorQuickOpenEntry {
 	}
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
-		if (mode === Mode.OPEN) {
-			const sideBySide = !context.quickNavigateConfiguration && context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
-			const pinned = !this.configurationService.getConfiguration<IWorkbenchEditorConfiguration>().workbench.editor.enablePreviewFromQuickOpen;
-
-			if (this.input instanceof EditorInput) {
-				this.editorService.openEditor(this.input, { pinned }, sideBySide).done(null, errors.onUnexpectedError);
-			} else {
-				this.editorService.openEditor({ resource: (this.input as IResourceInput).resource, options: { pinned: true } }, sideBySide);
```

Comment:
@wprater thanks, a bad bug that I introduced and will push a fix to master. it should be `{ pinned }` as the other one.